extensions/amp-kaltura-player/OWNERS

```diff
@@ -4,7 +4,11 @@
 {
   rules: [
     {
-      owners: [{name: 'itaykinnrot'}, {name: 'ampproject/wg-ui-and-a11y'}],
+      owners: [
+        {name: 'itaykinnrot'},
+        {name: 'ampproject/wg-ui-and-a11y'},
+        {name: '@mrhein'},
```

Comment:
It looks like @ is not allowed here. Could you pls remove?

Reply:
removed